packages/runtime/test-runtime-utils/src/mocks.ts

```diff
@@ -114,8 +114,8 @@ export class MockDeltaConnection implements IDeltaConnection {
 		this.handler?.processMessages?.(messageCollection);
 	}
 
-	public reSubmit(content: any, localOpMetadata: unknown) {
-		this.handler?.reSubmit(content, localOpMetadata);
+	public reSubmit(content: any, localOpMetadata: unknown, squash?: boolean) {
```

Comment:
i wanted to keep API changes to the mocks minimal since we have some known tech debt items to try and figure out what to do with this area.